nomadhook/lua/AI/AIBehaviors.lua

```diff
@@ -404,7 +404,7 @@ CommanderBombardPriorityList = {
 }
 
 function CommanderBombardThread(cdr, platoon)
-    if cdr:GetUnitId() ~= 'inu0001' then return end
+    if cdr:GetUnitId() ~= 'XNL0001' then return end
```

Comment:
I think this wont work, its case sensitive, some goes for the rest of your changes

Reply:
This depends on whether GetUnitId() sets everything to lower case I imagine. I can't see it being a problem, but if it breaks for whatever reason, I'll fix it.